samples/react-native/getting-started/app.config.js

```diff
@@ -30,4 +32,14 @@ export default {
     REACT_APP_PUB_KEY: process.env.REACT_APP_PUB_KEY,
     REACT_APP_SUB_KEY: process.env.REACT_APP_SUB_KEY,
   },
+  plugins: [
+    [
+      "expo-image-picker",
```

Comment:
Same here. We probably don't want to clutter the getting started app with a non-essential feature. Perhaps we could move these to the Group Chat app

Reply:
Actually now when I test that it seems that if I completely remove this section, expo-image-picker ends up with default values here. I'll remove it.